src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml

```diff
@@ -381,43 +430,201 @@
 
             <ui:PromptTextBox Margin="10,5" PromptText="Title" Style="{StaticResource GitHubVsPromptTextBox}" />
             <ui:PromptTextBox Margin="10,5" PromptText="Description" TextWrapping="Wrap" AcceptsReturn="True" Height="100" Style="{StaticResource GitHubVsPromptTextBox}" />
             <Grid Margin="10,0">
                 <Grid.ColumnDefinitions>
                     <ColumnDefinition Width="Auto" />
-                    <ColumnDefinition Width="*" />
                     <ColumnDefinition Width="Auto" />
+                    <ColumnDefinition Width="*" />
                 </Grid.ColumnDefinitions>
 
                 <Label Grid.Column="0" Content="Assignee" VerticalAlignment="Center" Foreground="{DynamicResource GitHubVsGrayText}" />
-                <ui:GitHubActionLink Grid.Column="1" Content="{Binding SelectedAssignee}" HasDropDown="True" VerticalAlignment="Center" HorizontalAlignment="Left">
-                    <i:Interaction.Behaviors>
-                        <ui:DropDownActionLinkBehaviour />
-                    </i:Interaction.Behaviors>
-
-                    <ui:GitHubActionLink.ContextMenu>
-                        <ContextMenu>
-                            <MenuItem Header="Haacked" />
-                            <MenuItem Header="NotHaacked" />
-                        </ContextMenu>
-                    </ui:GitHubActionLink.ContextMenu>
-                </ui:GitHubActionLink>
-
-                <ui:GitHubActionLink Grid.Column="2" Content="Clear Assignee" VerticalAlignment="Center" />
+                <ui:GitHubActionLink
+                    x:Name="assigneePopupButton" 
+                    Grid.Column="1"
+                    Content="Haacked (Phil Haack)"
+                    HasDropDown="True"
+                    VerticalAlignment="Center"
+                    HorizontalAlignment="Left"
+                    Click="assigneePopupButton_Click" />
+
+                <Popup
+                    x:Name="assigneePopup"
+                    AllowsTransparency="True"
+                    StaysOpen="False"
+                    PlacementTarget="{Binding ElementName=assigneePopupButton}"
+                    Placement="Bottom">
+                    <Border Style="{DynamicResource GitHubComboBoxBorder}">
+                        <Grid Style="{DynamicResource GitHubComboBoxContainer}"
+                            MinWidth="150"
+                            MaxWidth="300">
+                            
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="*" />
+                                <RowDefinition Height="Auto" />
+                            </Grid.RowDefinitions>
+
+                            <ui:FilterTextBox
+                                x:Name="filterAssignee"
+                                Grid.Row="0"
+                                Foreground="{DynamicResource GitHubVsWindowText}"
```

Comment:
I haz clue! Take out https://github.com/github/VisualStudio/blob/add-combobox-ui/src/GitHub.UI/Assets/Controls/FilterTextBox.xaml#L126, it's resetting the foreground in such a way that it overrides dynamic resource'd settings.